src/BlazingPizza.Client/Shared/LoginDisplay.razor

```diff
@@ -0,0 +1,17 @@
+﻿<div class="user-info">
```

Comment:
I'm assuming this is a rename to be more consistent with the server-side template. let me know if I missed something.

Reply:
That’s right.